lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_ntp_server
+short_description: Configures NTP server on Cisco UCS Manager
+extends_documentation_fragment:
+- ucs
+description:
+- Configures NTP server on Cisco UCS Manager.
+- Examples can be used with the L(UCS Platform Emulator,https://communities.cisco.com/ucspe).
+options:
+  state:
+    description:
+    - If C(absent), will remove an ntp server.
+    - If C(present), will add or update an ntp server.
+    choices: [absent, present]
+    default: present
+
+  name:
+    description:
+    - The ntp server ip address or hostname.
```

Comment:
I'd write this as `NTP server IP address or hostname`.

Reply:
Change made